Altdentifier/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+      "name": "NMW03",
+      "id": "534099893979971584"
+    },
+    "service": "Altdentifier",
+    "description": {
+      "en": "The best Discord security and checkpoint bot!"
+    },
+    "url": [
+      "altdentifier.com"
+    ],
+    "version": "1.0",
+    "logo": "https://altdentifier.com/static/cop.png",

```

Comment:
Please upload their logo to another site yourself. Imgur or your is-inside.me

Reply:
Done